nav2_costmap_2d/plugins/costmap_filters/keepout_filter.cpp

```diff
@@ -89,17 +89,18 @@ void KeepoutFilter::filterInfoCallback(
   if (msg->base != BASE_DEFAULT or msg->multiplier != MULTIPLIER_DEFAULT) {
     RCLCPP_ERROR(
       node->get_logger(),
-      "base and multiplier in ConstmapFilterInfo should be set to their default values (%d and %d)",
+      "For proper use of keepout filter base and multiplier in ConstmapFilterInfo "
```

Comment:
`ConstmapFilterInfo` -> `KeepoutFilter` (since this is the keepout filter :-) )

Reply:
This RCLCPP_ERROR is informing about base and multiplier in `CostmapFilterInfo` message, not about `CostmapFilter` or `KeepoutFilter` filter.
Also, "keepout filter" itself was also mentioned in the same sentence just before.
I think, I need to fix a typo instead: `ConstmapFilterInfo` -> `CostmapFilterInfo`.

Reply:
Ah I see, what I meant was to specifically say the words `keepout filter` so that people know this error is coming from the keepout filter. Because they could have many costmap filters and the `node->get_logger()` won't make that clear as to which one it is. But keeping the message name is fine, I think just adding the word `message` afterwards would be helpful. Just scanning past that quickly I didn't pick up on it meaning the message not the `costmap filter` base class itself.

Reply:
Hope, this is that you are looking for: https://github.com/ros-planning/navigation2/pull/1882/commits/19b3e114db484fad38a5e6e839a4c1f9bdbee303